hack/sdk/reviewable.sh

```diff
@@ -0,0 +1,10 @@
+# This script helps to lint and format the code in the SDK scaffold.
```

Comment:
```suggestion

#!/usr/bin/env bash

# Copyright 2022 The KubeVela Authors.
#
# Licensed under the Apache License, Version 2.0 (the "License");
# you may not use this file except in compliance with the License.
# You may obtain a copy of the License at
#
#     http://www.apache.org/licenses/LICENSE-2.0
#
# Unless required by applicable law or agreed to in writing, software
# distributed under the License is distributed on an "AS IS" BASIS,
# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
# See the License for the specific language governing permissions and
# limitations under the License.

set -o errexit
set -o nounset
set -o pipefail

# This script helps to lint and format the code in the SDK scaffold.
```

Make sure this script can exit with an error if any of the statements inside failed

Reply:
done